.eslintrc

```diff
@@ -0,0 +1,257 @@
+parser: "babel-eslint"
```

Comment:
You're missing a number of rules here - most notably the entire ES6 section. You should think about pulling in a sharable config here. I think there are going to be a LOT of thing to correct when you get those turned on :)